README.md

```diff
@@ -1,6 +1,21 @@
 # heart-prediction
 Heart attack risk prediction
 
+# Requirements installation
+To install requirements:
+$ python -m pip install -r requirements.txt
+
+# Installation of the created module heartpredictions
+To install all modules setup:
+$ python -m pip install -e .
+
 # TestSuite usage
-* Command to run the testsuite on linux: $ python -m pytest tests/*
-* If all tests passed then change the "actual_*.pkl" by the new one before a push
+* Command to run the testsuite on linux: 
+$ python -m pytest -v tests/Test*.py
+* Tests will fail as long as the weights giving a higher accuracy are not put in the best_weights folder
+
+# Test integration
+* Thanks to the python application github's action we add a tests integrations before a pull request merge and at each commit push
+
+# About the heartpredictions module
+TODO
```

Comment:
There is a TODO is it for this issue or another?

Reply:
Yes it is for the documentation issue (the 27th issue)